test/JDBC/expected/table_variable_xact_nested.out

```diff
@@ -541,11 +541,22 @@ int#!#text
 3#!#@table_variable_in_trigger_3
 3#!#@pg_toast_#oid_masked#
 3#!#@pg_toast_#oid_masked#_index
+3#!#inserted
+3#!#deleted
+3#!#@table_variable_in_trigger_2
+3#!#@pg_toast_#oid_masked#
+3#!#@pg_toast_#oid_masked#_index
+3#!#@tv_1
+3#!#@pg_toast_#oid_masked#
+3#!#@pg_toast_#oid_masked#_index
```

Comment:
why do we show more now? Also, are we supposed to show inserted and deleted? I think those are trigger related.

Reply:
The call to get_ENR now goes through parent queryEnvs, so any call to `sys.babelfish_get_enr_list()` will show all the TVs from upper nest level too, including those created by triggers